src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -297,24 +300,25 @@ def dump_exception(self, exception: Exception, bulk_results: BulkResult) -> None
         """
         # extract line run errors
         errors, line_runs = [], []
-        try:
-            for line_result in bulk_results.line_results:
-                if line_result.run_info.error is not None:
-                    errors.append(
-                        {
-                            "line number": line_result.run_info.index,
-                            "error": line_result.run_info.error,
-                        }
-                    )
-                line_runs.append(line_result)
-        except Exception:
-            pass
+        if bulk_results:
+            try:
+                for line_result in bulk_results.line_results:
+                    if line_result.run_info.error is not None:
+                        errors.append(
+                            {
+                                "line number": line_result.run_info.index,
+                                "error": line_result.run_info.error,
+                            }
+                        )
+                    line_runs.append(line_result)
+            except Exception:
+                pass
 
-        # won't dump exception if errors not found in bulk_results
-        if not errors:
-            return
+            # won't dump exception if errors not found in bulk_results
+            if not errors:
+                return
 
-        if exception is None:
+        if exception is None or not isinstance(exception, UserErrorException):
```

Comment:
Why we ignore system error in this PR?

Reply:
Because SystemError will be raised above and user can see it, so we don't need to dump it here.